package myerror

import (
	"fmt"
)

func HandleError(err error) {
	// TODO 改善
	fmt.Println(err)
}
